frontend/src/scenes/onboarding/personalizationLogic.ts

```diff
@@ -21,11 +24,20 @@ export const personalizationLogic = kea<personalizationLogicType>({
             },
         ],
     },
-    urlToAction: ({ actions }) => ({
-        '/personalization': (_: any, { step }: { step?: number | null }) => {
-            if (step && VALID_STEPS.includes(step)) {
-                actions.setStep(step)
-            }
+    listeners: {
+        reportPersonalizationSkipped: async () => {
```

Comment:
Unsure about migrating this logic from `eventUsageLogic`. The idea behind that file is centralizing and separating the logic of data instrumentation vs. core logic

Reply:
In this case the two are so coupled that IMO it makes sense to keep the logic together.